service/src/main/java/com/epam/esm/validator/realisation/certificate/DescriptionValidatorLink.java

```diff
@@ -0,0 +1,24 @@
+package com.epam.esm.validator.realisation.certificate;
+
+import com.epam.esm.dto.GiftCertificateDTO;
+import com.epam.esm.validator.PreparedValidatorChain;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class DescriptionValidatorLink extends PreparedValidatorChain<GiftCertificateDTO> {
+
+    private static final String REG_EXP_DESCRIPTION = "^[A-Za-z1-9 ]{1,}$";
+
+    @Override
+    public boolean validate(GiftCertificateDTO bean) {
```

Comment:
Looks a bit strange. Is it possible to have null dto in request? Moreover, does it make sense to pass it forward?